Open3Eclient.py

```diff
@@ -175,12 +175,11 @@ def cmnd_loop():
                         showread(addr=addr, did=itm[0], value=itm[1], idstr=itm[2])
 
                 elif cd['mode'] == 'write':
-                    # ToDo: Umrechnung über Codec ergänzen. Wechselwirkung mit flag_rawmode beachten!
                     addr = getaddr(cd)
                     ensure_ecu(addr)
                     for wd in cd['data']:
                         didKey = getint(wd[0])    # key: convert numeric or string parameter to numeric value
-                        didVal = getint(wd[1])    # value: dto.
+                        didVal = json.loads(wd[1])    # value: parse as json
```

Comment:
@MyHomeMyData This actually assumes that the data is as json-string inside a json. There are also use-cases where it would be a dictionary part of the encapsulating json. Should we enable both cases?

Reply:
> Should we enable both cases?

the more versatile the better!? (if not making it unreliable...)

Reply:
I think `json.loads` should cover all cases. We should keep to that.